arm-recoveryservices/2016-06-01/swagger/vaultusages.json

```diff
@@ -0,0 +1,188 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-06-01",
+    "title": "RecoveryServicesClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": false
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/usages": {

```

Comment:
I understand that the requirement seems to be to support one ListOperations API per swagger spec. We do have Operation_List APIs for each doc, but we have not yet exposed some of the APIs in Swagger yet. So, is it okay if the Operation_List API returns more APIs than that are defined in the spec?